### PR TITLE
Implement EIP 3607

### PR DESCRIPTION
**Pull request by OlivierBBB**
Reject transactions from senders with deployed code.

### CODE REVIEW

File: hub/constraints/tx_init/constraints.lisp [hunk 1]
```diff
@@ -58,6 +58,9 @@
                    (DOM-SUB-stamps---standard                     TX_INIT_SENDER_ACCOUNT_ROW_OFFSET
                                                                   0)))
 
+(defconstraint   tx-initialization---EIP-3607---reject-transactions-from-senders-with-deployed-codey          (:guard (tx-init---standard-precondition))
+                 (vanishes!    (shift    account/HAS_CODE    TX_INIT_SENDER_ACCOUNT_ROW_OFFSET)))
+
```
Comment (OlivierBBB) [inline hub/constraints/tx_init/constraints.lisp:63]: @letypequividelespoubelles here

File: hub/constraints/tx_skip/constraints.lisp [hunk 2]
```diff
@@ -29,156 +29,160 @@
...
+                 (account-same-code                            tx-skip---row-offset---sender-account)
+                 (account-same-deployment-number-and-status    tx-skip---row-offset---sender-account)
+                 (account-same-warmth                          tx-skip---row-offset---sender-account)
+                 (account-same-marked-for-selfdestruct         tx-skip---row-offset---sender-account)
+                 (account-isnt-precompile                      tx-skip---row-offset---sender-account)
+                 (DOM-SUB-stamps---standard                    tx-skip---row-offset---sender-account    0)))
+
+
+(defconstraint   tx-skip---EIP-3607---reject-transactions-from-senders-with-deployed-codey          (:guard (tx-skip---precondition))
+                 (vanishes!    (shift    account/HAS_CODE    tx-skip---row-offset---sender-account)))
```
Comment (OlivierBBB) [inline hub/constraints/tx_skip/constraints.lisp:59]: @letypequividelespoubelles here

File: hub/constraints/tx_skip/constraints.lisp [hunk 3]
```diff
@@ -11,16 +11,16 @@
...
 
 (defconst 
-  tx-skip---sender-account---row-offset       0
-  tx-skip---recipient-account---row-offset    1
-  tx-skip---coinbase-account---row-offset     2
-  tx-skip---transaction-row---row-offset      3)
+  tx-skip---row-offset---sender-account       0
+  tx-skip---row-offset---recipient-account    1
+  tx-skip---row-offset---coinbase-account     2
+  tx-skip---row-offset---transaction-row      3)
```
Comment (OlivierBBB) [inline hub/constraints/tx_skip/constraints.lisp:17]: Ignore this @letypequividelespoubelles : it's a more coherent renaming of row offset constants.

**Review by letypequividelespoubelles — APPROVED** [review]
LGTM

**Pull request merged**